src/module-vsbridge-indexer-core/Indexer/Action/Full.php

```diff
@@ -53,11 +62,13 @@ public function execute(array $ids)
             foreach ($stores as $store) {
                 $this->getIndexerHandler()->saveIndex($this->rebuild((int) $store->getId(), []), $store);
                 $this->getIndexerHandler()->cleanUpByTransactionKey($store);
+                $this->cacheProcessor->cleanCacheByTags($store->getId(), [$this->getIndexerHandler()->getTypeName()]);
             }
         } else {
             foreach ($stores as $store) {
                 $this->getIndexerHandler()->createIndex($store);
                 $this->getIndexerHandler()->saveIndex($this->rebuild((int) $store->getId(), []), $store);
+                $this->cacheProcessor->cleanCacheByTags($store->getId(), [$this->getIndexerHandler()->getTypeName()]);
```

Comment:
I think that you need add checking for
$cacheTags = $this-> cacheProcessor->getCacheTags();
isset($cacheTags[$this->getTypeName()];
and set to second params $cacheTags[$this->getIndexerHandler()->getTypeName()] because cleanCacheByTags method don't check cache tag value

Reply:
@sarron93 I added validation on cacheProcessor side https://github.com/vuestorefront/magento2-vsbridge-indexer/pull/369/files#diff-cacba1bef0b62da01a4252e13e6ad3da6560b711431bf6ed774cf085423fd541 please have a look